code/game/objects/items/mail.dm

```diff
@@ -215,19 +215,21 @@
 	name = "mail crate"
 	desc = "A certified post crate from CentCom."
 	icon_state = "mail"
+	base_icon_state = "mail"
 	can_install_electronics = FALSE
 	lid_icon_state = "maillid"
 	lid_x = -26
 	lid_y = 2
+	paint_jobs = null
 
 /obj/structure/closet/crate/mail/update_icon_state()
 	. = ..()
 	if(opened)
-		icon_state = "[initial(icon_state)]open"
+		icon_state = "[base_icon_state]open"
```

Comment:
This is because of the VV edit you suggested in your previous review for map loaded crates, 
![Screenshot (196)](https://user-images.githubusercontent.com/110812394/236454818-35603763-0e33-43f7-9079-df58885f4d62.png)
so for this crate the final icon state would be mailopen + open = mailopenopen. To prevent these edits from causing error's we use the base icon state instead